wdl/model/draft3/src/main/scala/wdl/model/draft3/elements/TaskBodyElement.scala

```diff
@@ -0,0 +1,3 @@
+package wdl.model.draft3.elements
+
+trait TaskBodyElement extends LanguageElement
```

Comment:
ToL: For these no-content traits, it might be a good idea to just have a one-liner comment above them. Eg we were just talking about this and I already had to think for a second to remember what this trait represents